easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.4.2-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '4.4.2'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://ffmpeg.org/releases/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['f98a482520c47507521a907914daa9efbc1384e0591b5afc3da18aa897de2948']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('pkg-config', '0.29.2')
```

Comment:
Can you change pkg-config-0.29.2 to pkgconf-1.8.0?

Reply:
Replaced with pkgconf